lib/src/main/cpp/README.md

```diff
@@ -1,3 +1,129 @@
 # Traditional keypoint detection algorithms
 
-TODO
+This directory contains the native implementation of the SGBM-based depth estimation algorithm.
+
+## Description
+
+[SIFT  (Scale-Invariant Feature Transform)](https://docs.opencv.org/3.4/d7/d60/classcv_1_1SIFT.html)
+
+[SURF (Speeded-Up Robust Features)](https://docs.opencv.org/3.4/d5/df7/classcv_1_1xfeatures2d_1_1SURF.html)
+
+[ORB (Oriented FAST and Rotated BRIEF)](https://docs.opencv.org/3.4/d7/d60/classcv_1_1SIFT.html)
+
+These algorithms from OpenCV is used to
+calculate keypoints and descriptors from image. The result is:
+Firsly, vector of structures which contains floating-point numbers representing each keypoint of picture
+Secondly, a vector of descriptors accounting each keypoint which is also a vector of a floating-point numbers
```

Comment:
Make it into bullets like this:
- Vector of structures ...
- Vector of descriptors ...